E_LEARNING.sql

```diff
@@ -0,0 +1,120 @@
+E-LEARNING PLATFORM
+
```

Comment:
Consider adding CreatedDate, UpdatedDate, and CreatedBy columns to relevant tables